test/dockertest.go

```diff
@@ -6,19 +6,19 @@ import (
 	"github.com/ory/dockertest/v3"
 )
 
-//This will create a docker container using the RunOptions given
+//CreateContainer will create a docker container using the RunOptions given
 //
 //"opts" is the configuration for docker
 //
 //"retryOp" is an exponential backoff-retry, because the application in the container might not be ready to accept connections yet
-func CreateContainer(opts dockertest.RunOptions, retryOp func(r *dockertest.Resource) error) (err error, purgeFn func() error) {
+func CreateContainer(opts dockertest.RunOptions, retryOp func(r *dockertest.Resource) error) (purgeFn func() error, err error) {
```

Comment:
@scortier All integration tests needs to be refactored for this. You can try that by running with integration tag. 
cc @GrayFlash

Reply:
yup, so @scortier we need to go through all the tests in plugins/extractors and interchange position of output variables this function returns, and test it using `go test -tags=integration`
for e.g.;
err, purgeFn := 
becomes
purgefn, err :=

Reply:
got it 👍